app/views/feedbacks/index.html.slim

```diff
@@ -0,0 +1,34 @@
+h1= t_title
+
+table.table.table-striped
+  thead
+    tr
+      th= t_model(Volunteer)
+      th= t_attr(:feedback_activity, Feedback)
+      th= t_attr(:goals, Feedback)
+      th= t_attr(:achievements, Feedback)
+      th= t_attr(:future, Feedback)
+      th= t_attr(:comments, Feedback)
+      th= t_attr(:conversation, Feedback)
+      th colspan='3'
+
+  tbody
+    - @feedbacks.each do |record|
+      tr
+        td= link_to record.volunteer.contact.full_name, volunteer_path(record.volunteer)
+        td
+          - if record.assignment?
+            = link_to record.feedbackable.to_label, client_path(record.feedbackable.client)
+          - else
+            = link_to record.feedbackable.to_label, group_offer_path(record.feedbackable)
```

Comment:
Note that there's also `polymorphic_path`, maybe you can use it here or in other places. See http://thelazylog.com/polymorphic-routes-in-rails/

Reply:
yep, the paths are all quite a bit of a lacky hacky thingie here.